docs/general/server/media/music.md

```diff
@@ -125,6 +125,7 @@ Jellyfin supports most common music formats, with some exceptions:
 - WebA: `.weba` files aren't supported. Please rename them to `.mka`
 - `.flac` files with embedded WebP images may fail to play in Chromium based browsers (Chrome, Edge, Opera, Brave etc.).
 - `.flac` files with ID3 tags won't play in Firefox.
+- Files with ID3v1 tags have a length limit of 30 bytes for most fields. Anything longer will be truncated. Please upgrade to ID3v2.
```

Comment:
```suggestion
- Files with ID3v1 tags have a length limit of 30 bytes for most fields. Anything longer will be truncated. Please upgrade to ID3v2.4.
```

Just to make sure people don't start using v2.3 with custom multi tag implementations...